src/commands/rm.ts

```diff
@@ -24,17 +24,31 @@ interface RmFlags {
 
 async function executeRemove(cwd: string, args: string[]) {
   const flags = parseArgs(args);
-  await Promise.all(flags.paths.map((specifiedPath) => {
-    if (specifiedPath.length === 0) {
-      throw new Error("Bug in dax. Specified path should have not been empty.");
-    }
-    const path = resolvePath(cwd, specifiedPath);
-    if (path === "/") {
-      // just in case...
-      throw new Error("Cannot delete root directory. Maybe bug in dax? Please report this.");
+  try {
+    await Promise.all(flags.paths.map((specifiedPath) => {
+      if (specifiedPath.length === 0) {
+        throw new Error(
+          "Bug in dax. Specified path should have not been empty.",
+        );
+      }
+
+      const path = resolvePath(cwd, specifiedPath);
+      if (path === "/") {
+        // just in case...
+        throw new Error(
+          "Cannot delete root directory. Maybe bug in dax? Please report this.",
+        );
+      }
+
+      return Deno.remove(path, { recursive: flags.recursive });
```

Comment:
I believe it should just be a try/catch around this `Deno.remove` only so it doesn't exit if one of the paths fails?

Maybe just:

```ts
return Deno.remove(path, { recursive: flags.recursive }).catch(err => {
  if (flags.force && err instanceof Deno.errors.NotFound) {
    return Promise.resolve();
  } else {
    return Promise.reject(err);
  }
});

Reply:
I tried the try/catch around the `Deno.remove` and the exception still bubbled up. Your snippet with `catch` looks promising and worth a try though. I'll also see about adding a test.

Thanks for this project, BTW. I needed something like `shx` for Deno in our code generation project called [Apex](https://apexlang.io) and this fits the bill!

Reply:
That project looks really useful.